### PR TITLE
Feature/simplify disk req bsc#1118795

### PR DESCRIPTION
**Pull request by taroth21**
### Description
This is about https://bugzilla.suse.com/show_bug.cgi?id=1118795.

### Checklist
* Check all items that apply.

*Are backports required?*

- [ ] To maintenance/SLE12SP3
- [ ] To maintenance/SLE12SP4
- [x] To maintenance/SLE15SP0

### CODE REVIEW

**Comment by taroth21** [comment]
PDF of the current version:
 [cha.x86_color_draft_en.pdf](https://github.com/SUSE/doc-sle/files/3023417/cha.x86_color_draft_en.pdf)

File: xml/deployment_prep_x86_nsysreqs.xml [hunk 1]
```diff
@@ -72,119 +72,55 @@
...
+     The hard disk requirements do not only depend on the selected product and
+     the packages installed, but also on how you use your machine. In addition
+     to the space occupied by the software itself, you need some more space for
+     a system to work properly.
     </para>
-    <informaltable>
+    <para>
+     Especially if you want to enable Btrfs snapshots on the root volume to
+     enable system rollbacks (see <xref linkend="cha.snapper"/>), the amount of
+     hard disk space is crucial. The more snapshots you want to store and older
```
Comment (sknorr) [inline xml/deployment_prep_x86_nsysreqs.xml]: Seems like this sentence needs an extra "the". Suggestion: "The more snapshots you want to store and *the* older..."

File: xml/deployment_prep_x86_nsysreqs.xml [hunk 2]
```diff
@@ -72,119 +72,55 @@
...
-        <entry>
-         <para>
-          Minimal System
-         </para>
-        </entry>
+        <row>
         <entry>
          <para>
-          2.5 GB
+         10 GB (Required Minimum)
```
Comment (sknorr) [inline xml/deployment_prep_x86_nsysreqs.xml]: Intentation seems off by one space now.

File: xml/deployment_prep_x86_nsysreqs.xml [hunk 3]
```diff
@@ -72,119 +72,55 @@
...
          </para>
         </entry>
-       </row>
-       <row>
         <entry>
          <para>
-          &gnome; Desktop
+         16 GB (Required Minimum)
```
Comment (sknorr) [inline xml/deployment_prep_x86_nsysreqs.xml]: Intentation seems off by one space now.

File: xml/deployment_prep_x86_nsysreqs.xml (hunk 1 above)
Comment (taroth21) [reply xml/deployment_prep_x86_nsysreqs.xml]: Argh, good catch!

**Comment by taroth21** [comment]
@sknorr : Thank you for a second pair of eagle eyes. :)

**Comment by taroth21** [comment]
Closing unmerged as of https://bugzilla.suse.com/show_bug.cgi?id=1118795#c9.

**Pull request closed without merging**